docs-source/content/userguide/managing-domains/fmw-infra/_index.md

```diff
@@ -0,0 +1,343 @@
+running in pods in Kubernetes to see the database.  This can be done by defining a
+Kubernetes service with no selector and associating it with an endpoint definition, as shown
+in the example below: 
+
+```yaml
+kind: Service
+apiVersion: v1
+metadata:
+ name: database
+spec:
+ type: ClusterIP
+ ports:
+ - port: 1521
+   targetPort: 1521
+---
+kind: Endpoints
+apiVersion: v1
+metadata:
+ name: database
+subsets:
+ - addresses:
+     - ip: 129.123.1.4
+   ports:
+     - port: 1521
+```
+
+This creates a DNS name `database` in the current namespace, or`default` if no namespace is 
+specified, as in the example above.  The fully qualified name would be
+`database.default.svc.cluster.local` in this example.  The second part is the namespace.
+If you looked up the `ClusterIP` for such a service, it would have an IP address on the overlay
+network, that is the network inside the Kubernetes cluster.  If you are using flannel
+for example, the address might be something like `10.0.1.25`.  Note that this is usually a
+non-routed address.
+
+From a container in a pod running in Kubernetes, you can make a connection to that address
+and port `1521`.  Kubernetes will route the connection to the address provided in the
+endpoint definition, in this example `129.123.1.4:1521`.  This IP address (or name) is
+resolved from the point of view of the Kubernetes node's IP stack, not the overlay network
+inside the Kubernetes cluster.  Note that this is a "real" routed IP address. 
+
+When you create your data sources, you would use the internal address, for example
+`database:1521/some.service`.
+
+Because your database is externally accessible, you can run RCU in the normal way, from any
+machine on your network.
+
+#### Running the Repository Creation Utility to set up your database schema
+
+If you want to run RCU from a pod inside the Kubernetes cluster, you can use the Docker
+image that you built earlier as a "service" pod to run RCU.  To do this, start up a
+pod using that image as follows: 
+
+```bash
+kubectl run rcu -ti --image oracle/fmw-infrastructure:12.2.1.3 -- sleep 100000
+
+```
+
+This will create a Kubernetes deployment called `rcu` containing a pod running a container
+created from the `oracle/fmw-infrastructure:12.2.1.3` image which will just run
+`sleep 100000`, which essentially creates us a pod that we can "exec" into and use to run whatever
+commands we need to run. 
+
+To get inside this container and run commands, use this command: 
+
+```bash
+kubectl exec -ti rcu /bin/bash
+```
+
+When you are finished with this pod, you can remove it with this command: 
+
+```bash
+kubectl delete deploy rcu 
+```
+
+{{% notice note %}}
+You can use the same approach to get a temporary "service" pod to run other utilities
+like WLST.
+{{% /notice %}}
+
+##### Creating schemas
+
+Inside this pod, you can use the following command to run RCU in command line (no GUI) mode
```

Comment:
command line (no GUI) mode -> command-line (no GUI) mode (hyphenate command line when it is used as an adj)

Reply:
fixed